apps/coscrad-frontend/src/components/resources/factories/table-view-categorizable-index-presenter-factory.tsx

```diff
@@ -30,9 +30,7 @@ const ConcreteSpatialFeaturePresenter = (
 ): JSX.Element => (
     <SpatialFeatureIndexPresenter
         MapComponent={CoscradLeafletMap}
-        DetailPresenter={thumbnailCategorizableDetailPresenterFactory(
-            CategorizableType.spatialFeature
-        )}
+        DetailPresenter={fullViewCategorizablePresenterFactory(CategorizableType.spatialFeature)}
```

Comment:
Ah- that was the problem. The thumbnail view is the one we use on top of the map markers. Good catch!